dashboard/.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.{js,jsx,ts,tsx,vue}]
+indent_style = space
```

Comment:
Is it possible to also add semicolons required and single quotes only? Does this work with vscode? Does this conflict with prettier?

Reply:
This configures IDE settings across editors. In VS Code you need to install a plugin. Those formatting rules are included by the linter automatically using the airbnb style guide. It's _very_ strict